angular-portfolio/src/app/awards/awards.component.scss

```diff
@@ -0,0 +1,52 @@
+.awards {
+    background-color: #ecf0f1;
+    height: 65vh;
+
+    .flex-container {
+        align-items: center;
+        flex-direction: row;
+        display: flex;
+        position: relative;
+
+        .years {
+            flex-direction: columns;
```

Comment:
Is this working properly? I believe it should be "flex-direction: column;"

Reply:
Thank you! Compilation did not pick this up.